plugins/generic/pln/classes/DepositPackage.inc.php

```diff
@@ -84,6 +84,27 @@ function getAtomDocumentPath() {
 	 function getPackageFilePath() {
 		return $this->getDepositDir() . DIRECTORY_SEPARATOR . $this->_deposit->getUUID() . ".zip";
 	}
+    
+    /**
+     * Create a DOMElement in the $dom, and set the element name, namespace, and 
+     * content appropriately. Named HTML entities in the content are converted
+     * to UTF-8 characters. Any invalid UTF-8 characters will be dropped.
+     * 
+     * @param DOMDocument $dom
+     * @param string $elementName
+     * @param string $content
+     * @param string $namespace
+     * @return DOMElement
+     */
+	function _generateElement($dom, $elementName, $content, $namespace = null){
+		$encoded = html_entity_decode($content, ENT_NOQUOTES | ENT_HTML401, 'UTF-8');
+		$filtered = iconv('UTF-8', 'UTF-8//IGNORE', $encoded);
```

Comment:
`iconv` (historically at least) has not always been available; it might be safer to use `mb_convert_encoding` per e.g. http://stackoverflow.com/questions/8215050/replacing-invalid-utf-8-characters-by-question-marks-mbstring-substitute-charac